### ISSUE TITLE
Milliseconds converted incorrectly to seconds

### ISSUE THREAD

**Issue by Elijen** [fmarcia/UglifyCSS#31]
Units in ms are incorrectly converted to seconds:
```css
/* test.css */
.animated {
  animation-duration: 600ms;
}
```
```css
/* result of `uglifycss test.css` */
.animated{animation-duration:60s}
```
Expected output:
```css
.animated{animation-duration:600ms}
```

**Comment by Elijen** [comment]
Quick investigation revealed that 6fd4970271345b89633c6131bc506dbddeebfa53 broke this.

**Comment by Elijen** [comment]
Seems this has already been fixed by 453ec83a0ccc0199d0f057b8ec065da9a656b3ec but not published to NPM yet.

**Comment by fmarcia** [comment]
... as you said ;-)
